### ISSUE TITLE
Support for DynamoDb Streams?

### ISSUE THREAD

**Issue by caindy** [theburningmonk/ReactoKinesix#60]
Would adding the ability to process DynamoDb Streams be a welcomed addition to the capabilities of this library? AWS provides [an adapter](https://github.com/awslabs/dynamodb-streams-kinesis-adapter) allowing the Java KCL to be used with DynamoDb Streams; it seems like a natural fit for ReactoKinesix. Would you be kind enough to share thoughts on what the right extension points would be?

**Comment by theburningmonk** [comment]
Makes sense to add support for it, I need to familiarize myself with how their adapter works and see what the right approach is for us here. Thanks for bringing it up, let me think on it.
